documentation/userguide.md

```diff
@@ -95,12 +95,39 @@ After restarting the IIS/recycling the pool and opening the application in a bro
 
 ### Azure Cloud
 
-Azure instantly kills the application when it is stopped/restarted, thus the profiler has no time to write anything to the trace file. To remedy this, enable eager mode (see the Configuration section. Please note that eager mode should only be used together with light mode as it will otherwise drastically increase lag during the application startup.
-
-To deploy the profiler to the Azure cloud, enable the FTP account. Store the profiler in a directory that will not be overwritten by a new deployment, e.g. `D:\home\site\repository\profiler\`
-Also create a new directory to write the traces to, e.g. `D:\home\site\repository\traces\`
-
-To set the environment variables in Azure, go to the application settings and add them as new application settings entries.
+If the application is running in an Azure App Service, the following steps need to be taken:
+
+**Install the Profiler:**
+To deploy the profiler, [enable the FTP account](https://docs.microsoft.com/en-us/azure/app-service/app-service-deploy-ftp) and upload the profiler DLLs to a location that will not be overwritten by a new app deployment, e.g., `D:\home\site\repository\profiler\`.
+To [configure the profiler](#profiler_configuration) in Azure, go to the [application settings](https://docs.microsoft.com/en-us/azure/app-service/web-sites-configure) and add the environment variables as new application settings entries.
+For the profiler target directory, you may use `D:\home\LogFiles` or a dedicated location, such as `D:\home\site\repository\profiler\traces` (remember to manually create this directory first).
+
+**Install the UplaodDaemon:**
+Deploy the upload daemon in the directory `.\UploadDaemon`, relative to the path where you deployed the profiler, e.g., in `D:\home\site\repository\profiler\UploadDaemon\`.
+[Configure the trace upload](#automatic_trace_upload) as usual.
+
+**Schedule Trace Upload:**
+By default, the profiler publishes traces when the app gets shut down (unless you [configure "Eagerness"(#profiler_configuration)).
+Azure Apps are shut down automatically [in cases of user inactivity or at least every 29 hours](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-application-and-service-availability-faq#why-does-the-first-request-to-my-cloud-service-after-the-service-has-been-idle-for-some-time-take-longer-than-usual).
+To use a different intervall, you can configure a different schedule in `D:\home\site\applicationHost.xdt`:
+
+    <?xml version="1.0"?>
+    <configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+      <system.applicationHost>
+        <applicationPools>
+          <add name="yourappname" xdt:Locator="Match(name)"> // <-- fill app name
```

Comment:
not a valid XML comment. Please use an XML comment as people are prone to just copy-paste stuff and expect it to work. This will give them weird parsing errors